webpack.config.js

```
@@ -1,3 +1,5 @@
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

ESLint

[error] Definition for rule '@typescript-eslint/no-var-requires' was not found.

(@typescript-eslint/no-var-requires)